kfdefs/overlays/moc/smaug/seldon/alerts.yaml

```diff
@@ -0,0 +1,37 @@
+apiVersion: monitoring.coreos.com/v1
+kind: PrometheusRule
+metadata: 
+  name: seldon-alerts
+spec: 
+  groups: 
+    - 
+      name: alert_rules
+      rules: 
+        - alert: InstanceDown
+          expr: up {namespace="opf-seldon"}== 0
+          for: 1m
```

Comment:
maybe increase this to 10m or 15m
as some time, deployment is just syncing up or restarting 
so lets open alerts if for a period if its down